bam_readcount_helper.py

```diff
@@ -35,11 +32,7 @@ def filter_sites_in_hash(region_list, bam_file, ref_fasta, sample, output_dir, i
     else:
         sys.exit(stderr)
 
-min_mapping_qual = None
-if len(sys.argv) == 7:
-    (script_name, min_base_qual, vcf_filename, sample, ref_fasta, bam_file, output_dir)= sys.argv
-elif len(sys.argv) == 8:
-    (script_name, min_mapping_qual, min_base_qual, vcf_filename, sample, ref_fasta, bam_file, output_dir)= sys.argv
+(script_name, vcf_filename, sample, ref_fasta, bam_file, output_dir, min_mapping_qual, min_base_qual)= sys.argv
```

Comment:
I think I wasn't quite clear how this should be changed. You basically have to handle three cases:
1) `len(sys.argv) == 6` i.e, backwards compatibility where neither mapping quality nor base quality values are passed along
2) `len(sys.argv) == 7` i.e., when only min_base_qual is passed along
3) `len(sys.argv) == 8` i.e, both min_base_qual and min_mapping_qual are provided. Additionally, to be consistent with case (2) above, you'll want to provide the min_base_qual before the min_mapping_qual.
For cases (1) and (2) you will also set those variables that are not passed along to a default.